src/hotspot/share/jfr/jni/jfrJavaSupport.cpp

```diff
@@ -343,6 +343,8 @@ static void write_specialized_field(JfrJavaArguments* args, const Handle& h_oop,
     case T_BOOLEAN:
     case T_CHAR:
     case T_SHORT:
+      Unimplemented();
+      break;
```

Comment:
Why not implement these, while we are at it?

Reply:
because it's more work, and we probably "Ain't Gonna Need It". The method is 6 years old, and is only used in one place.
That being said, if you think these types might be useful in a predictable future, I can implement them.

Reply:
Let's implement them. As it stands now, they are "implemented", albeit incorrectly.